### PR TITLE
Updated German translation

### PR DESCRIPTION
**Pull request by Marvbell110**

### CODE REVIEW

File: locales/de.lua [hunk 1]
```diff
@@ -8,93 +8,93 @@ Locales['de'] = {
   ['give_to'] = 'geben an',
   ['amount'] = 'betrag',
   ['giveammo'] = 'munition geben',
-  ['amountammo'] = 'anzahl der munition',
```
Comment (ElPumpo) [inline locales/de.lua]: im curious to why did you change that `munition` word to upper case?

Comment (Marvbell110) [reply locales/de.lua]: In german it  munition upper case

File: locales/de.lua [hunk 2]
```diff
@@ -8,93 +8,93 @@ Locales['de'] = {
...
-  ['threw_weapon_already'] = 'Du hast bereits diese Waffe',
-  ['threw_cannot_pickup'] = 'Du kannst das nicht aufheben, da dein Inventar voll ist',
-  ['threw_pickup_prompt'] = 'drücke ~y~E~s~ um aufzuheben',
+  ['amountammo'] = 'anzahl der Munition',
+  ['noammo'] = 'du hast keine Munition!',
+  ['gave_item'] = 'du gibst ~y~%sx~s~ ~b~%s~s~ an ~y~%s~s~.',
+  ['received_item'] = 'du empfängst ~y~%sx~s~ ~b~%s~s~ von ~b~%s~s~.',
+  ['gave_weapon'] = 'du gibst ~b~%s~s~ an ~y~%s~s~.',
+  ['gave_weapon_ammo'] = 'du gibst ~o~%sx %s~s~ von ~b~%s~s~ an ~y~%s~s~.',
+  ['gave_weapon_withammo'] = 'du gibst ~b~%s~s~ mit ~o~%sx %s~s~ an ~y~%s~s~.',
```
Comment (ElPumpo) [inline locales/de.lua]: and here why have you added dots to each string? no

File: locales/de.lua [hunk 3]
```diff
@@ -8,93 +8,93 @@ Locales['de'] = {
...
+  ['received_weapon_noweapon'] = '~b~%s~s~ hat versucht dir Munition für eine(n) ~y~%s~s~, aber du besitzt diese Waffe nicht.',
+  ['gave_account_money'] = 'du gibst ~g~$%s~s~ (%s) an ~y~%s~s~.',
+  ['received_account_money'] = 'du empfängst ~g~$%s~s~ (%s) von ~b~%s~s~.',
+  ['amount_invalid'] = 'ungültiger Betrag!',
+  ['players_nearby'] = 'keine Spieler in der Nähe!',
+  ['ex_inv_lim'] = 'aktion nicht möglich, Inventarlimit überschritten für ~y~%s~s~.',
+  ['imp_invalid_quantity'] = 'aktion nicht möglich, ungültige Anzahl!',
+  ['imp_invalid_amount'] = 'aktion nicht möglich, ungültiger Betrag!',
+  ['threw_standard'] = 'du wirfst ~y~%sx~s~ ~b~%s~s~ weg!',
+  ['threw_account'] = 'du wirfst ~g~$%s~s~ ~b~%s~s~ weg!',
```
Comment (ElPumpo) [inline locales/de.lua]: and here you use ! marks but they shouldnt be used here, they have a purpose and this is not one of them

File: locales/de.lua [hunk 4]
```diff
@@ -8,93 +8,93 @@ Locales['de'] = {
...
+  ['received_weapon_hasalready'] = '~b~%s~s~ hat versucht dir eine(n) ~y~%s~s~ zu geben, aber du hast bereits eine(n).',
+  ['received_weapon_noweapon'] = '~b~%s~s~ hat versucht dir Munition für eine(n) ~y~%s~s~, aber du besitzt diese Waffe nicht.',
+  ['gave_account_money'] = 'du gibst ~g~$%s~s~ (%s) an ~y~%s~s~.',
+  ['received_account_money'] = 'du empfängst ~g~$%s~s~ (%s) von ~b~%s~s~.',
+  ['amount_invalid'] = 'ungültiger Betrag!',
+  ['players_nearby'] = 'keine Spieler in der Nähe!',
+  ['ex_inv_lim'] = 'aktion nicht möglich, Inventarlimit überschritten für ~y~%s~s~.',
+  ['imp_invalid_quantity'] = 'aktion nicht möglich, ungültige Anzahl!',
+  ['imp_invalid_amount'] = 'aktion nicht möglich, ungültiger Betrag!',
+  ['threw_standard'] = 'du wirfst ~y~%sx~s~ ~b~%s~s~ weg!',
```
Comment (ElPumpo) [inline locales/de.lua]: so get the ! characters removed where they dont belong

**Comment by dutchplayers** [comment]
This is not Dutch, but German!!!

**Comment by ElPumpo** [comment]
Haha didnt know that sorry

**Comment by ElPumpo** [comment]
Re open if you ever want to fix the mistakes
